amgut/lib/data_access/ag_data_access.py

```diff
@@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
         return return_res
 
     def getAnimalParticipants(self, ag_login_id):
-        results = self._sql.execute_proc_return_cursor(
-            'ag_get_animal_participants', [ag_login_id])
-
-        return_res = [row[0] for row in results]
-        results.close()
-        return return_res
+        sql = """SELECT participant_name FROM ag_animal_survey
```

Comment:
That is from the old table, yes. Do you want me to migrate everything over as part of this patch?